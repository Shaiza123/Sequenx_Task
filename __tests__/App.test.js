// /**
//  * @format
//  */

// import 'react-native';
// import React from 'react';
// import App from '../App';
// import renderer from 'react-test-renderer'

// test('render correctly', () => {
//   const snapshot = renderer.create(<App />).toJSON();
//   expect(snapshot).toMatchSnapshot();
// })
