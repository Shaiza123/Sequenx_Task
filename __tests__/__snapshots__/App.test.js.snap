// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hi
  </Text>
</View>
`;
